appveyor-nightly.yml

```diff
@@ -26,6 +26,8 @@ environment:
   KEEP_VIGNETTES: true
   R_ARCH: x64
   R_VERSION: 4.1.0
+  #We use this variale to skip some long lasting tests using "skip_on_ci"
+  CI: true
```

Comment:
@Yuri05 @PavelBal @IndrajeetPatil In the furture, if we need to turn off those tests, we just need to do this

Reply:
Nice! 👍